src/Employee.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Objects;
+
+public class Employee {
+    private String nameFull;
+    private int numDepart;
+    private float salary;
+    private int id;
+    static int i = 1;
```

Comment:
лучше назвать как-то более понятно, например, так
```suggestion
    static int idGenerator = 1;
```